.github/workflows/coverage-check.yml

```diff
@@ -33,8 +33,6 @@ jobs:
         id: ui-changes
         uses: tj-actions/changed-files@a59f800cbb60ed483623848e31be67659a2940f8
         with:
-          sha: "${{ github.pull_request.sha }}"
-          base_sha: "${{ github.sha }}"
```

Comment:
Due to the way `pull_request_target` works, when explicit sha, base_sha aren't passed I saw it fails to diff well. Maybe the values aren't right, something else should be passed?